CMakeLists.txt

```diff
@@ -367,6 +388,8 @@ if(WITH_VTK AND NOT ANDROID)
       # safe to assume OpenGL backend
       set(VTK_RENDERING_BACKEND "OpenGL")
     endif()
+    list(APPEND PCL_VTK_COMPONENTS vtkRenderingContext${VTK_RENDERING_BACKEND})
+    find_package(VTK COMPONENTS ${PCL_VTK_COMPONENTS})
     message(STATUS "VTK_MAJOR_VERSION ${VTK_MAJOR_VERSION}, rendering backend: ${VTK_RENDERING_BACKEND}")
```

Comment:
`include(${VTK_USE_FILE})` will be only called in case `(PCL_SHARED_LIBS OR (NOT (PCL_SHARED_LIBS) AND NOT (VTK_BUILD_SHARED_LIBS)))` returns true., Shouldn't it be always? (You removed some other calls to this include, so it could be an issue now)

Reply:
> E.g. HAVE_VTK is undefined in case WITH_VTK is true, but VTK was not found.

I think this is okay because undefined variables evaluate to false in conditionals.